### ISSUE TITLE
Jet Forms Builder – Reviews

### ISSUE THREAD

**Issue by DanielShuster** [CrocoBlock/suggestions#6737]
Great idea to add reviews system integrated to jet forms builder. (like Jet Reviews)

Why? Because jet forms builder have a lot of options that can be integrated in one. 
I have a problem in jet reviews and over a month no one can help me, I've tried: support in website, telegram, Facebook community, GitHub (here) and no one can help.

Moreover, there are issues with the CSS in jet reviews (star won't change colors etc…) the team said it's known issue.

Thanks :)

**Comment by Lonsdale201** [comment]
+1

**Comment by jabedkhan** [comment]
Yeah, I also want to this functionality. If we can able to create review form then we can add diffrent diffrent style in the review listing.
